main.js

```diff
@@ -0,0 +1,206 @@
+'use strict';
+
+let gl;                         // The webgl context.
+let surface;                    // A surface model
+let shProgram;                  // A shader program
+let spaceball;                  // A SimpleRotator object that lets the user rotate the view by mouse.
+
+
+function deg2rad(angle) {
+    return angle * Math.PI / 180;
+}
+
+
+// Constructor
+function Model(name) {
+    this.name = name;
+    this.iVertexBuffer = gl.createBuffer();
+    this.count = 0;
+
+    this.BufferData = function(vertices) {
+
+        gl.bindBuffer(gl.ARRAY_BUFFER, this.iVertexBuffer);
+        gl.bufferData(gl.ARRAY_BUFFER, new Float32Array(vertices), gl.STREAM_DRAW);
+
+        this.count = vertices.length/3;
+    }
+
+    this.Draw = function() {
+
+        gl.bindBuffer(gl.ARRAY_BUFFER, this.iVertexBuffer);
+        gl.vertexAttribPointer(shProgram.iAttribVertex, 3, gl.FLOAT, false, 0, 0);
+        gl.enableVertexAttribArray(shProgram.iAttribVertex);
+        
+        gl.drawArrays(gl.LINE_STRIP, 0, this.count);
```

Comment:
Remove the line inside the figure limiting this line of code

Reply:
![image](https://user-images.githubusercontent.com/45340638/201132336-90d66fcc-e616-4fb4-ac16-f1a2a410fc3a.png)